code/modules/gear_presets/survivors/solaris/preset_solaris.dm

```diff
@@ -79,9 +79,25 @@
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/jackboots(new_human), WEAR_FEET)
 	..()
 
+/datum/equipment_preset/survivor/uscm/solaris
+	name = "Survivor - Solaris United States Colonial Marines Corps Recruiter"
```

Comment:
Survivor - Solaris United States Colonial Marine Corps Recruiter

Typo

Reply:
Where exactly is the typo? The s in Corp?

Reply:
Marines should be Marine

Reply:
Done